### PR TITLE
Fix fwd & inverse compatibility issues.

### LINKED ISSUE #191: read_forward_solution getting deprecated arg
@larsoner `read_forward_solution()` getting deprecated arg `surf_ori`. Will try fixing with `convert_forward_solution()`

### PR DESCRIPTION
**Pull request by ktavabi**
@larsoner please review and confirm loose/free/fixed source ori parameters. Fixes #191

### CODE REVIEW

File: mnefun/_mnefun.py [hunk 1]
```diff
@@ -1639,9 +1640,14 @@ def gen_inverses(p, subjects, run_indices):
             for f, m, e in zip(out_flags, meg_bools, eeg_bools):
                 fwd_restricted = pick_types_forward(fwd, meg=m, eeg=e)
                 for l, s, x, d in zip(looses, tags, fixeds, depths):
+                    if l < 1:
+                        use_cps = True
+                    else:
+                        use_cps = False
```
Comment (larsoner) [inline mnefun/_mnefun.py]: we should always use CPS

File: mnefun/_mnefun.py [hunk 2]
```diff
@@ -1621,13 +1621,14 @@ def gen_inverses(p, subjects, run_indices):
...
             temp_name = s_name + ('-%d' % p.lp_cut) + p.inv_tag
             fwd_name = op.join(fwd_dir, s_name + p.inv_tag + '-fwd.fif')
-            fwd = read_forward_solution(fwd_name, surf_ori=True)
-            looses = [None]
+            fwd = read_forward_solution(fwd_name)
+            fwd = convert_forward_solution(fwd, surf_ori=True)
+            looses = [0]
             tags = [p.inv_free_tag]
-            fixeds = [False]
+            fixeds = [True]
```
Comment (larsoner) [inline mnefun/_mnefun.py]: False

File: mnefun/_mnefun.py [hunk 3]
```diff
@@ -1621,13 +1621,14 @@ def gen_inverses(p, subjects, run_indices):
             s_name = safe_inserter(name, subj)
             temp_name = s_name + ('-%d' % p.lp_cut) + p.inv_tag
             fwd_name = op.join(fwd_dir, s_name + p.inv_tag + '-fwd.fif')
-            fwd = read_forward_solution(fwd_name, surf_ori=True)
-            looses = [None]
+            fwd = read_forward_solution(fwd_name)
+            fwd = convert_forward_solution(fwd, surf_ori=True)
+            looses = [0]
```
Comment (larsoner) [inline mnefun/_mnefun.py]: [1]

File: mnefun/_mnefun.py (hunk 1 above)
Comment (ktavabi) [reply mnefun/_mnefun.py]: So `else use_cps = None` for the fixed and free ori cases, correct?

Comment (larsoner) [reply mnefun/_mnefun.py]: I would just always set `use_cps=True`

Comment (ktavabi) [reply mnefun/_mnefun.py]: The [docstr](https://martinos.org/mne/dev/generated/mne.minimum_norm.make_inverse_operator.html) for `make_inverse_operator` says cps is...

>Only used when converting to surface orientation (i.e., for surface source spaces and loose < 1).

Won't it balk with `loose = 1`?

**Comment by larsoner** [comment]
Does it work okay with `use_cps=True` always on?

**Comment by ktavabi** [comment]
> Does it work okay with use_cps=True always on?

Looks that way. It's not balking in the pipeline for funloc!

**Comment by larsoner** [comment]
Thanks for taking care of these!

**Pull request merged**